packages/react-provider/package.json

```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "graphql": "^15.5.0",
     "react-use": "^17.2.1",
+    "reconnecting-websocket": "^4.4.0",
```

Comment:
this one was last updated 1 year ago?

Reply:
looks like it yeah

Reply:
it also looks slightly heavy. but is it getting tree shaken? I am mostly curious if adding this dependency bloats our package or not.

Reply:
@suhasdeshpande this package is 2.6kb minified and gzipped https://bundlephobia.com/package/reconnecting-websocket@4.4.0

Reply:
It's also tree shakeable and has no dependencies.

![Screen Shot 2021-06-30 at 3 28 52 PM](https://user-images.githubusercontent.com/266012/124039493-e4744600-d9b7-11eb-838b-cb04a7caa0cd.png)

Reply:
curious, what made you think it was slighly heavy?   i don't see any indication of that.

Reply:
I didn't realize I was looking at unpacked size on npm. 
<img width="460" alt="Screen Shot 2021-06-30 at 3 50 35 PM" src="https://user-images.githubusercontent.com/606167/124041043-f9060d80-d9ba-11eb-9589-81e645f69a5e.png">

Reply:
ahh, isee

Reply:
Unpacked size can be deceiving. Mobx is 3.84 MB unpacked but like 16.1kB minified and gzipped